test/e2e_node/criproxy_test.go

```diff
@@ -68,7 +68,13 @@ var _ = SIGDescribe(feature.CriProxy, framework.WithSerial(), func() {
 			framework.ExpectNoError(err)
 
 			pod := e2epod.NewPodClient(f).Create(ctx, newPullImageAlwaysPod())
-			podErr := e2epod.WaitForPodRunningInNamespace(ctx, f.ClientSet, pod)
```

Comment:
We inject an error and the pod will never be running. And the timeout is 10 minutes here.

I changed the logic to just wait for a failure pod with timeout 1 minute.